repos/system_upgrade/el7toel8/actors/spamassassinconfigcheck/actor.py

```diff
@@ -0,0 +1,23 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.models import Report, SpamassassinFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class SpamassassinConfigCheck(Actor):
+    """
+    No documentation has been provided for the spamassassin_config_check actor.
```

Comment:
Please don't forget to update the documentation

Reply:
I'm working on it as we speak :).